drivers/irqchip/irq-bcm2836.c

```diff
@@ -175,7 +175,11 @@ __exception_irq_entry bcm2836_arm_irqchip_handle_irq(struct pt_regs *regs)
 		u32 ipi = ffs(mbox_val) - 1;
 
 		writel(1 << ipi, mailbox0);
+#ifdef CONFIG_ARM64
+                dsb(sy);
```

Comment:
Can you check your editor settings and make sure you match indentation with surrounding code.
Generally the linux kernel uses tabs for indentation. You can use ``scripts/checkpatch.pl`` to check for obvious formatting violations. Don't worry about making the existing code clean, but try not to add new complaints.